python/lsst/afw/table/arrays.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from ._arrays import *
```

Comment:
I think we're going to have to introduce a new exception in the conventions for this sort of case; it's filling much the same role as an `__init__.py` file.